airbyte-integrations/bases/base-typing-deduping/src/main/java/io/airbyte/integrations/base/destination/typing_deduping/AirbyteType.java

```diff
@@ -107,17 +83,57 @@ private static Array getArray(final JsonNode schema) {
     }
   }
 
+  private static AirbyteType fromArrayJsonSchema(final JsonNode schema, final JsonNode array) {
```

Comment:
I think we're at a point where all these inner classes should just get their own file... feels like having them all in one file is doing more harm than good